src/backend/distributed/utils/citus_readfuncs.c

```diff
@@ -331,6 +357,38 @@ ReadTask(READFUNC_ARGS)
 
 
 READFUNC_RET
+ReadTaskExecution(READFUNC_ARGS)
+{
+	READ_LOCALS(TaskExecution);
+	int count = 0;
+
+	READ_UINT64_FIELD(jobId);
```

Comment:
If you want I think it'd be ok to replace the read function with an error throwing one - outputting is useful for debugging, but there's little reason to allow deserialization.